requirements.txt

```diff
@@ -1,39 +1,46 @@
+#
+# These requirements were autogenerated by pipenv
+# To regenerate from the project's Pipfile, run:
+#
+#    pipenv lock --requirements
+#
+
```

Comment:
I think this header is generated by the new version of Pipenv right? Would you mind setting up a new PR that upgrades the pipenv version?

Reply:
Yes it's from the new pipenv version, but I don't think there's a way we can specify pipenv in our dependencies.

Reply:
Oh you're right. As long as everything works with the new version, then we should be fine.